data/core/encyclopedia/geography.cfg

```diff
@@ -50,7 +50,19 @@ The central island of the archipelago is also called ‘Morogor’."
 [topic]
     id=green_isle
     title= _ "Green Isle"
-    text= _ "A bigger island lying in the <ref>dst='great_ocean' text='Great Ocean'</ref>."
+    text= _ "A bigger island lying in the <ref>dst='great_ocean' text='Great Ocean'</ref>, home to the Islefolk and Wesfolk.
```

Comment:
Was home to the Islefolk and Wesfolk, prior to the events in The Rise of Wesnoth.

I think that's no more of a spoiler than the campaign description itself.